### PR TITLE
Matrix: Feedback column doesn't need to show #744929

### PR DESCRIPTION
**Pull request by mkassaei**
Hi @timhunt, Thanks for the support this morning. I have changed this accordingly and ready for Review.

### CODE REVIEW

**Comment by timhunt** [comment]
The think is, we are not removing the unnecsesary feedback column. We are just changing the title to an empty string.

That does not seem great to me. E.g. accessibility. Are we sure?

File: question.php [hunk 1]
```diff
@@ -168,6 +168,20 @@ public function update_attempt_state_data_for_new_version(
 
         return $startdata;
     }
+
+    /**
+     * Check whether question has any specific feedback.
+     *
+     * @return bool
+     */
+    public function has_specific_feedback_for_row(): bool {
```
Comment (timhunt) [inline question.php]: Please we rename this to "has_specific_feedback". I think that is clearer.

**Review by timhunt — CHANGES_REQUESTED** [review]
Thanks Mahmoud. That looks good now. Please can I be picky and ask you to rename one method? Thanks.

File: question.php (hunk 1 above)
Comment (mkassaei) [reply question.php]: Hi @timhunt, thanks for the review. I have renamed has_specific_feedback_for_row() and pushed the patch.

**Review by timhunt — APPROVED** [review]

**Comment by timhunt** [comment]
Thanks. Merged.

**Pull request merged**